src/pages/rewards/RewardsPage.tsx

```diff
@@ -191,61 +92,25 @@ const Styled: Record<string, AnyStyledComponent> = {};
 Styled.Page = styled.div`
   ${layoutMixins.contentContainerPage}
   gap: 1.5rem;
-
-  @media ${breakpoints.tablet} {
-    padding: 1rem;
-  }
```

Comment:
<img width="1094" alt="Screen Shot 2023-10-24 at 10 39 29 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/26fe5683-0759-46d9-810a-3a0c39df8297">

<img width="926" alt="Screen Shot 2023-10-24 at 10 44 54 AM" src="https://github.com/dydxprotocol/v4-web/assets/13111994/8d9fd81f-55d9-409b-8e94-ce0605e18f91">

Add min padding on SmallDesktop/Tablet/Mobile. I think the logic for calculated padding works for Desktop when we are using the `<WithSidebar>` but it seems off w/o it. Not sure if it looks funny because I haven't defined the migration link

Reply:
was planning to fix this in another PR